### DISCUSSION TITLE
ElementGroup Layout property missing

### DISCUSSION THREAD

**Discussion by Enzogord** [HandyOrg/HandyControl#1155] [Q&A] [answered]
Hi I'm trying to use ElementGroup but this component is missing a Layout property. In the sources it has this property. 
And in the sources class ElementGroup based on ItemsControl, but from nuget he based on SimpleStackPanel.
What is the correct way to use ElementGroup?

**Comment by ghost1372** [comment, marked as answer]
Nuget package is not updated yet( corrently use old source code which layout property is not exist.) You should wait for next nuget release and layout property will be available for you.

**Reply by Enzogord** [reply]
Thank you.
